### PR TITLE
Dockerize

### LINKED ISSUE #16: Self hostable
Options are: 
* dockerize
* scripts to set up file env
* document self host process

### PR DESCRIPTION
**Pull request by MrPicklePinosaur**
closes #16 

also need to write some docs still - and test if it acc works

### CODE REVIEW

**Comment by fairnightzz** [comment]
While you're at it, can you also make sure the github action workflow dockerizes instead? Additionally might be super excessive, but we could have 2 workflows and 2 branches - one for development, master for prod.

**Comment by MrPicklePinosaur** [comment]
> While you're at it, can you also make sure the github action workflow dockerizes instead? Additionally might be super excessive, but we could have 2 workflows and 2 branches - one for development, master for prod.

do you want a `stage`/`next` branch? and then once we're sure its working fine we move to master?

**Comment by fairnightzz** [comment]
That's what I meant by `development` branch yeah, I don't know about `next` but `stage` is fine

**Comment by MrPicklePinosaur** [comment]
okidoke

**Comment by MrPicklePinosaur** [comment]
should get rid of the `docker-compose`

**Comment by fairnightzz** [comment]
Why get rid of the docker compose?

**Comment by MrPicklePinosaur** [comment]
move it to `tonkatsu`

File: Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM node:lts-alpine
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
```
Comment (MrPicklePinosaur) [inline Dockerfile.prod:3]: ?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ React webapp that can read zipped manga files from url. Built for KaratsubaScans
 
 ## Running for development
 
+Docker:
+
+```
+docker-compose up
```
Comment (MrPicklePinosaur) [inline README.md:10]: when giving code examples for shell commands, i like to place a prompt character to indicate that it's a command
```
$ docker-compose up
```

File: docker-compose.prod.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+version: "3.8"
+services:
+  app:
+    container_name: carrot
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    expose:
+      - 80
```
Comment (MrPicklePinosaur) [inline docker-compose.prod.yml:10]: also good practice to add
```
app:
  restart: always
```
to automatically restarted dead containers

**Review by MrPicklePinosaur — COMMENTED** [review]
lgtm

**Review by fairnightzz — APPROVED** [review]

**Pull request merged**